build/ndm-daemonset/Dockerfile.in

```diff
@@ -9,10 +9,4 @@ ARG ARCH
 
 #Copy binary to /usr/sbin/ndm
 COPY bin/${ARCH}/ndm /usr/sbin/ndm
-COPY build/ndm-daemonset/entrypoint.sh /usr/local/bin/entrypoint.sh
-
-#Expose port 9090
-EXPOSE 9090
-
-#Set the default command
-ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
+COPY build/ndm-daemonset/entrypoint.sh /usr/local/bin/entrypoint
```

Comment:
keep the .sh suffix.

Reply:
Done. Also kept the entrypoint as is, only `args` will be passed not the command.
The `args` will be passed as arguments to the entrypoint script.